### DISCUSSION TITLE
Add a particular filter to video + display sequencially the text

### DISCUSSION THREAD

**Discussion by bourbe** [Zulko/moviepy#1937] [Q&A]
Hello, 

I want to produce a video like this with moviepy with the same filter effect and the same display of the text

I will display them essentially on a 1080 x 1920 screen

[https://www.youtube.com/shorts/bt1AispMJQI](url)

1. wchich fx I have to use to have this result please ?
2. How can I code the sequentially display of the text centered like the one in the video
3. Do you have an idea of some elegant font I can use or the one used in this  video ?

Regards

**Comment by keikoro** [comment]
Please open a proper issue for this (using our issue templates) instead of using GitHub's Discussions feature, thank you.
